ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/ApiActiveContractsService.scala

```diff
@@ -17,26 +17,25 @@ import com.digitalasset.ledger.api.v1.active_contracts_service._
 import com.digitalasset.ledger.api.v1.event.CreatedEvent
 import com.digitalasset.ledger.api.validation.TransactionFilterValidator
 import com.digitalasset.platform.api.grpc.GrpcApiService
-import com.digitalasset.platform.common.logging.NamedLoggerFactory
 import com.digitalasset.dec.DirectExecutionContext
 import com.digitalasset.platform.participant.util.LfEngineToApi
 import com.digitalasset.platform.server.api.validation.ActiveContractsServiceValidation
 import io.grpc.{BindableService, ServerServiceDefinition}
+import org.slf4j.LoggerFactory
 import scalaz.syntax.tag._
 
 import scala.concurrent.ExecutionContext
 
 class ApiActiveContractsService private (
     backend: ACSBackend,
-    parallelism: Int = Runtime.getRuntime.availableProcessors,
-    loggerFactory: NamedLoggerFactory)(
+    parallelism: Int = Runtime.getRuntime.availableProcessors)(
     implicit executionContext: ExecutionContext,
     protected val mat: Materializer,
     protected val esf: ExecutionSequencerFactory)
     extends ActiveContractsServiceAkkaGrpc
     with GrpcApiService {
 
-  private val logger = loggerFactory.getLogger(this.getClass)
+  private val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
Why not use a contextualized logger here and in the other `*Service` classes?

Reply:
I mentioned this in the PR description, but thanks for asking, so that I can go in more detail about it.

It's a bit complicated, but I'm working on it: all services that extend `LoggingServiceMarker` automatically generate code that allows them to log gRPC internal and unknown errors. This requires the class `logger` to be overridden which worked fine with the `NamedLoggerFactory` but requires some refitting to work with the implicit `LoggingContext`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/e993b246d3aae04c81db4701d482a94aa75e31b1